Doc/library/crypt.rst

```diff
@@ -38,6 +38,8 @@ this module.
 
 .. availability:: Unix, not VxWorks.
 
+.. include:: ../includes/wasm-notavail.rst
+
```

Comment:
Not super important because this is a deprecated module with removal scheduled in 3.13, but this does lead to a duplicated `availability` directive, which is not ideal.